src/frontend/handlers.go

```diff
@@ -72,6 +78,11 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
+	//get env and render correct platform banner.
+	var env = os.Getenv("ENV_PLATFORM")
+	plat := platformDetails{}
+	plat.setPlatformDetails(strings.ToLower(env))
```

Comment:
nit/nbd: you can move strings.ToLower to the `setPlatformDetails` method for clarity:

    env = strings.ToLower(env)

and maybe add Azure there while at it?

Reply:
done